### DISCUSSION TITLE
SQLTablesW returns

### DISCUSSION THREAD

**Discussion by kothapraveen** [FreeTDS/freetds#536] [Q&A]
Hi @freddy77 

Currenlty we are trying to gettables info using  SQLTablesW using fakeid as CatalogName, SchemaName, TableName, TableType

SQL Logs:
[ODBC][146978][1710313398.330156][SQLTablesW.c][392]
                Exit:[SQL_ERROR]
                DIAG [08004] [FreeTDS][SQL Server]Database 'fakeid' does not exist. Make sure that the name is entered correctly.

FreeTDS Logs:

odbc.c:4259:SQLFreeStmt(0x1806230, 0)
odbc.c:4176:_SQLFreeStmt(0x1806230, 0, 0)
odbc_export.h:1255:SQLTablesW(0x1806230, fakeid, 6, fakeid, 6, fakeid, 6, fakeid, 6)
odbc.c:6912:SQLTables(0x1806230, 0x154eba0, 6, 0x182dca0, 6, 0x154eca0, 6, 0x181cb50, 6)
odbc.c:6977:to_fix 1 elements 1
odbc.c:6981:fixing type elements
odbc.c:3286:_SQLExecute(0x1806230)

currently we are facing below exception
Exception: (911, 08004, '[mtodbc]: Fetching table info: [FreeTDS][SQL Server]Database 'fakeid' does not exist. Make sure that the name is entered correctly.')

But when we execute Same source code in windows, we did not see this exception.

**Comment by freddy77** [comment]
Does `fakeid.fakeid.fakeid` table exist ?
